index.html

```diff
@@ -22,24 +22,160 @@
+            <li><a href="#">css</a></li>
+            <li><a href="#">javaScript</a></li>
+          </ul>
+          <div class="view-3">
+            <a href="#"><img src="images/Enabled.jpg"></a>
+          </div>
+        </footer>
+    </div>
+  </div>
 
+  <div class="page-4">
+    <div class="container-4">
+      <a href="#"><img src="images/Snapshoot Portfolio 3.jpg"></a>
+      <div id="text-4">
+        <a href="#"><img src="images/Project Title.jpg"></a>  
+      </div>
+        <ul class="column-4">CANOPY
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/back.jpg"></a></li>
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/year.jpg"></a></li>
+        </ul>
+      <div>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <footer>
+       <ul class="work-4">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul> 
+        <div class="view-4">
+          <a href="#"><img src="images/Enabled.jpg"></a>
+        </div>
+      </footer>
+    </div>
+  </div>
+  <div class="page-5">
+    <div class="container-5">
+      <a href="#"><img src="images/Snapshoot Portfolio 4.jpg"></a>
+      <div id="text-5">
+        <a href="#"><img src="images/Project Title.jpg"></a>  
+      </div>
+        <ul class="column-5">CANOPY
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/back.jpg"></a></li>
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/year.jpg"></a></li>
+        </ul>
+      <div>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <footer>
+       <ul class="work-5">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul> 
+        <div class="view-5">
+          <a href="#"><img src="images/Enabled.jpg"></a>
+        </div>
+      </footer>
+    </div>
+  </div>
 
+  <div class="page-6">
+    <section class="about-myself">
+      <h1>About Myself</h1>
+      <p>Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+      <a href="#">LET'S CONNECT</a>
+      <ul class="social-icons-2">
+        <li><a href="#"><img src="images/Twitter.jpg" alt="twitter"></a></li>
+        <li><a href="#"><img src="images/Linkedin icon.jpg" alt="Linkedin"></a></li>
+        <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
+        <li><a href="#"><img src="images/Githhub.jpg" alt="Github"></a></li>
+      </ul>
```

Comment:
Kindly modify the about myself section to mathch the design as closely as possible.

Your design:
![image](https://user-images.githubusercontent.com/20554583/199732534-e070d39e-4a0e-4edf-88b8-11f243b02411.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/199732363-e9ef1072-caa3-469d-9efb-01d741d0b049.png)